solvers/skglm.py

```diff
@@ -8,11 +8,13 @@
     from skglm.penalties import L1
     from skglm.solvers import cd_solver_path
     from sklearn.exceptions import ConvergenceWarning
+    from sklearn.linear_model._base import _preprocess_data
 
 
 class Solver(BaseSolver):
     name = "skglm"
     stopping_strategy = "iteration"
+    support_sparse = False
```

Comment:
you can implement a `skip` method to skip only if sparse.issparse(X) and fit_intercept (because skglm supports sparse without intercept)

Reply:
True, just realized it when doing the modifs for benchmark_lasso! Thanks pointing it out